cli/lib/otel/index.ts

```diff
@@ -1,15 +1,19 @@
-import { CollectorTraceExporter, CollectorMetricExporter } from '@opentelemetry/exporter-collector'
+import { JaegerExporter } from '@opentelemetry/exporter-jaeger'
+import { PrometheusExporter } from '@opentelemetry/exporter-prometheus'
 import { HttpInstrumentation } from '@opentelemetry/instrumentation-http'
 import { WinstonInstrumentation } from '@opentelemetry/instrumentation-winston'
-import { Resource, envDetector, processDetector } from '@opentelemetry/resources'
+import { Resource } from '@opentelemetry/resources'
 import { NodeSDK } from '@opentelemetry/sdk-node'
 import { SemanticResourceAttributes } from '@opentelemetry/semantic-conventions'
 
+const jaegerExporter = new JaegerExporter()
+const prometheusExporter = new PrometheusExporter()
```

Comment:
Use the OTLP exporter instead. Leaving all settings will make it use the various env variables to set the endpoint & co

```suggestion
const otlpMetricExporter = new OTLPMetricExporter();
```